tests/test_nonci.py

```diff
@@ -29,7 +30,7 @@ def test_launcher_output(docker_nonci):
         f"bash -c 'mkdir {cwd}/artifacts; echo \"Old artifact\" > {cwd}/artifacts/test_nonci.txt'")
 
     docker_nonci.project_root = None
-    console_out_log = docker_nonci.run(config.format(cwd), workdir=cwd)
+    console_out_log = docker_nonci.run(config.format(cwd), workdir=cwd, force_installed=True)
```

Comment:
The reason of these changes is not clear from the PR.
I guess you can write it into the commit message, but as of now I don't understand it.

Reply:
Updated the description of the PR.

Reply:
Ok, the PR description looks nice, but the intention of this exact line (and similar ones) is still not clear.